### PR TITLE
[examples] Fixes to ERC20 contract

### PR DESCRIPTION
**Pull request by shawntabrizi**
Fixes https://github.com/paritytech/ink/issues/73

### CODE REVIEW

File: examples/lang/erc20/src/lib.rs [hunk 1]
```diff
@@ -257,7 +258,7 @@ mod tests {
         let bob = AccountId::try_from([0x1; 32]).unwrap();
         assert_eq!(erc20.total_supply(), 1234);
         assert_eq!(erc20.balance_of(alice), 1234);
-        assert_eq!(erc20.transfer_from(alice, bob, 234), true);
```
Comment (Robbepop) [inline examples/lang/erc20/src/lib.rs]: What is bad about `transfer_from` here? I think it is more explicit.

Comment (shawntabrizi) [reply examples/lang/erc20/src/lib.rs]: Cause then we never test the `transfer` function.

`transfer_from` is tested when you do the allowance stuff

Comment (shawntabrizi) [reply examples/lang/erc20/src/lib.rs]: I want to actually also add a failed test

Comment (Robbepop) [reply examples/lang/erc20/src/lib.rs]: Sounds fair!

Comment (Robbepop) [reply examples/lang/erc20/src/lib.rs]: With failure test you mean a test that `panics` or a test that returns `false` upon `transfer` ?

Comment (shawntabrizi) [reply examples/lang/erc20/src/lib.rs]: Just added. A test that tries to transfer more than Alice has, and returns false.

**Pull request merged**